hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/opcodes/Create2OperationSuite.java

```diff
@@ -638,8 +641,10 @@ final Stream<DynamicTest> canMergeCreate2MultipleCreatesWithHollowAccount() {
         final AtomicReference<AccountID> partyId = new AtomicReference<>();
         final AtomicReference<ByteString> partyAlias = new AtomicReference<>();
 
-        return defaultHapiSpec("CanMergeCreate2MultipleCreatesWithHollowAccount", FULLY_NONDETERMINISTIC)
+        return propertyPreservingHapiSpec("CanMergeCreate2MultipleCreatesWithHollowAccount", FULLY_NONDETERMINISTIC)
+                .preserving("entities.unlimitedAutoAssociationsEnabled")
                 .given(
+                        overriding("entities.unlimitedAutoAssociationsEnabled", FALSE),
```

Comment:
⚠️  I think `propertyPreservingHapiSpec()` is failing to preserve the default value of `false` from your earlier tests that override it to `true`.

Probably because `propertyPreservingHapiSpec()` relies on [this file](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/test-clients/src/main/resource/bootstrap.properties#L57) to find the default value of a property; and right now we don't have the `entities.unlimitedAutoAssociationsEnabled=false` line.

So I think:
  1. Add the default to that file; and,
  2. Delete all the places that you have a spec **beginning** by overriding it to `false`.

This should work imo. 🤔 I created [this issue](https://github.com/hashgraph/hedera-services/issues/13931) to fix the underlying issue with `propertyPreservingHapiSpec()`.

Reply:
Agreed! I've updated the suites accordingly.